modules/my-module/android/src/main/java/expo/modules/mymodule/MyModule.kt

```diff
@@ -32,8 +38,19 @@ class MyModule : Module() {
 
     // Defines a JavaScript function that always returns a Promise and whose native code
     // is by default dispatched on the different thread than the JavaScript runtime runs on.
+    // TODO: this is the other side of the bridge!
     AsyncFunction("setValueAsync") { value: String ->
       // Send an event to JavaScript.
+
+      val rescheduleConfirmations = Json.decodeFromString<List<JsRescheduleConfirmationObject>>(value)
+
+      Log.i(TAG, rescheduleConfirmations.take(3).toString())
+      
+      // Send intent with reschedule confirmations data
+      val intent = Intent("com.github.quarck.calnotify.RESCHEDULE_CONFIRMATIONS")
+      intent.putExtra("reschedule_confirmations", value)
+      appContext.reactContext?.sendBroadcast(intent)
```

Comment:
Honestly, this is more of a celebration than a comment. I can't believe it's just work right on the first try 😂